tf-modules/cumulus/archive.tf

```diff
@@ -23,6 +23,7 @@ module "archive" {
   elasticsearch_security_group_id           = var.elasticsearch_security_group_id
   elasticsearch_remove_index_alias_conflict = var.elasticsearch_remove_index_alias_conflict
 
+  sf_event_sqs_lambda_timeout = var.sf_event_sqs_lambda_timeout
```

Comment:
Should we update the example terraform? I always ask this and the answer is usually "it's not intended to exemplify all possible options". I think it's fine not to, just raising it for kicks.

Reply:
The main benefit as I see it would just be documentation if users are looking at the example config. It's probably best to have it well documented in the variables tf like you have it.

Reply:
If by example, you mean `/example`, I tend to agree that that's really not meant to be an example to *users*.      There's a reasonable case to be made that we should have *module* documentation, but that's out of scope.